deployments/polygonzkevm.diamond.immutable.json

```diff
@@ -0,0 +1,48 @@
+{
+  "LiFiDiamondImmutable": {
+    "Facets": {
+      "0xF1d67E1dddc87E2858C87EBd5b19f99a4E297541": {
+        "Name": "DiamondCutFacet",
+        "Version": "1.0.0"
+      },
+      "0x49d195D3138D4E0E2b4ea88484C54AEE45B04B9F": {
+        "Name": "DiamondLoupeFacet",
+        "Version": "1.0.0"
+      },
+      "0x44beA2Ab010d1CedC4E60E97DA8F88b8840951B0": {
+        "Name": "OwnershipFacet",
+        "Version": "1.0.0"
+      },
+      "0x428C4abf8BB3ECc893bD3E0Be12b0f363c6e81aA": {
+        "Name": "WithdrawFacet",
+        "Version": "1.0.0"
+      },
+      "0x64D41a7B52CA910f4995b1df33ea68471138374b": {
+        "Name": "DexManagerFacet",
+        "Version": "1.0.0"
+      },
+      "0xfaA5f885a54D22C8571806fC001E53F0191f5Aff": {
+        "Name": "AccessManagerFacet",
+        "Version": "1.0.0"
+      },
+      "0x2EfC66F1ff37fc5277De5526Ab5CB7650f2DD518": {
+        "Name": "PeripheryRegistryFacet",
+        "Version": "1.0.0"
+      },
+      "0xA1Edc2eD671Dfa77eD2dCD2ee012F82e4807A75a": {
+        "Name": "CBridgeFacet",
+        "Version": "1.0.0"
+      }
+    },
+    "Periphery": {
```

Comment:
Please check if it is correct/desired that no periphery has been added to the diamond. It seems incorrect.